users/views.py

```diff
@@ -189,3 +189,25 @@ def get(self, request, pk=None, format=None):
 
         serializer = self.get_serializer(page, many=True)
         return self.get_paginated_response(serializer.data)
+
+
+class ClaimPrizeResource(APIView):
```

Comment:
perhaps we are missing some permissions here. This endpoint should require auth and the caregiver role (my understanding is that a caregiver can claim a prize on behalf of a student)